bigbluebutton-html5/imports/ui/components/modals/settings/submenus/SessionMenu.jsx

```diff
@@ -1,17 +1,70 @@
 import React from 'react';
 import Modal from 'react-modal';
-import Icon from '/imports/ui/components/icon/component';
 import Button from '/imports/ui/components/button/component';
 import BaseMenu from './BaseMenu';
+import { callServer } from '/imports/ui/services/api';
+import Auth from '/imports/ui/services/auth';
+import Storage from '/imports/ui/services/storage';
+import styles from '../../styles';
+import classNames from 'classnames';
+
+const customModal = {
+  overlay: {
+    zIndex: 2000,
+  },
+  content: {
+    width: '25%',
+    height: '18%',
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+  },
+};
 
 export default class SessionMenu extends BaseMenu {
   constructor(props) {
     super(props);
+    this.state = {
+      openConfirm: true,
+    };
+    this.closeLogout = this.closeLogout.bind(this);
+  }
+
+  closeLogout() {
+    this.setState({ openConfirm: false });
+  }
+
+  logout() {
```

Comment:
The component shouldn't use server functions `callServer`. You problably should have logout function on the `Auth` service and just call it here and then you could also reuse that function on the `onError` method from the App Service.